docs/publish/centralized-deployment.md

```diff
@@ -45,38 +45,37 @@ If your organization meets all requirements, complete the following steps to pub
+    - **I want to add an Add-In from the Office Store.**
     - **I have the manifest file (.xml) on this device.** For this option, choose **Browse** to locate the manifest file (.xml) that you want to use.
     - **I have a URL for the manifest file.** For this option, type the manifest's URL in the field provided.
 
     ![New Add-In dialog in Office 365 admin center](../images/new-add-in.png)
 
-7.	Choose **Next**.
-
-8.	If you selected the option to add an add-in from AppSource, select the add-in. Notice that you can view available add-ins via categories of **Suggested for you**, **Rating**, or **Name**. You may only add free add-ins from AppSource; adding paid add-ins isn't currently supported.
+8.	If you selected the option to add an add-in from Office Store, select the add-in. You can view available add-ins via categories of **Suggested for you**, **Rating**, or **Name**. You may only add free add-ins from Office Store. Adding paid add-ins isn't currently supported.
 
     > [!NOTE]
-    > With the AppSource option, updates and enhancements to the add-in will automatically be made available to users without your intervention.
+    > With the Office Store option, updates, and enhancements to the add-in are automatically available to users without your intervention.
 
     ![Select an Add-In dialog in Office 365 admin center](../images/select-an-add-in.png)
 
-9. The add-in is now available for assignment. On the page for the add-in, its status is **On**, like that shown for the Power BI Tiles add-in in the screenshot below. In the **Who has access** section, choose **Edit** to assign the add-in to users and/or groups.
+9. Choose **Next** after reviewing the add-in details.
 
     ![Power BI Tiles add-in page in Office 365 admin center](../images/power-bi-tiles.png)
 
-10.	On the **Edit who has access page**, choose either **Everyone** or **Specific Users/Groups**. Use the Search box to find the users and/or groups to whom you want to deploy the add-in.
+10.	On the **Edit who has access** page, choose **Everyone**, **Specific Users/Groups**, or **Only me**. Use the search box to find the users and groups to whom you want to deploy the add-in.
 
     ![Edit who has access page in Office 365 admin center](../images/power-bi-tiles-edit.png)
 
     > [!NOTE]
     > For single sign-on (SSO) add-ins, the users and groups assigned will also be shared with add-ins that share the same Azure App ID. Any changes to user assignments will also apply to those add-ins. The related add-ins will be shown on this page. For SSO add-ins only, this page will display the list of Microsoft Graph permissions that the add-in requires.
 
-11.	When finished, choose **Save**, review the add-in settings, and then choose **Close**. You now see your add-in along with other apps in Office 365.
+11.	The manifest then saves, which may take up to three minutes. Then, finish the walkthrough by pressing **Next**. You now see your add-in along with other apps in Office 365.
```

Comment:
When does the user press **Save**?  Seems like the instructions should specify this instruction somewhere...maybe at the start of this item (11)?